hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/state/DispatchingEvmFrameState.java

```diff
@@ -138,9 +143,9 @@ public void setStorageValue(final long number, @NonNull final UInt256 key, @NonN
      */
     @Override
     public @NonNull List<StorageAccesses> getStorageChanges() {
-        final Map<Long, List<StorageAccess>> modifications = new TreeMap<>();
```

Comment:
Question: is it ok to use HashMap instead of TreeMap here? I made the change because TreeMap wants a Comparable key and ContractID is not Comparable.

Reply:
Fixed